doc/man/8/cephadm.rst

```diff
@@ -527,6 +529,12 @@ Arguments:
 * [--name NAME, -n NAME]  daemon name (type.id)
 
 
+list-images
+-----------
+
+List the default container images for all services
```

Comment:
We might want to add to this description that we list them in an ini format so the results can be modified and passed to the cluster with the `--config` flag during bootstrap.